zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/znode/groupacl/ZkClientUriDomainMappingHelper.java

```diff
@@ -36,7 +42,7 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * An implementation of ClientUriDomainMappingHelper that stores the mapping inside the ZK server
+ * An implementation of ZkClientUriDomainMappingHelper that stores the mapping inside the ZK server
```

Comment:
Since `ClientUriDomainMappingHelper` is the interface this class `implements`, I'm not sure why we need to change this?